acs-integration-tests/src/test/resources/shared-test-spring-context.xml

```diff
@@ -1,74 +1,80 @@
-   
-    Unless required by applicable law or agreed to in writing, software 
-    distributed under the License is distributed on an "AS IS" BASIS, 
-    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
-    See the License for the specific language governing permissions and 
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
     limitations under the License.
  -->
 
 <beans xmlns="http://www.springframework.org/schema/beans"
-    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:context="http://www.springframework.org/schema/context"
-    xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-4.1.xsd
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xmlns:context="http://www.springframework.org/schema/context"
+       xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-4.1.xsd
     http://www.springframework.org/schema/context http://www.springframework.org/schema/context/spring-context-4.1.xsd">
 
+    <context:annotation-config/>
+    <context:component-scan base-package="com.ge.predix.test"/>
+
     <bean
-        class="org.springframework.beans.factory.config.PropertyPlaceholderConfigurer"
-        xmlns="http://www.springframework.org/schema/beans">
+            class="org.springframework.beans.factory.config.PropertyPlaceholderConfigurer"
+            xmlns="http://www.springframework.org/schema/beans">
         <property name="locations">
             <list>
                 <value>classpath:application.properties</value>
             </list>
         </property>
     </bean>
 
-    <bean id="zacRestTemplate"
-        class="org.springframework.security.oauth2.client.OAuth2RestTemplate">
-        <constructor-arg>
-            <bean
-                class="org.springframework.security.oauth2.client.token.grant.client.ClientCredentialsResourceDetails">
-                <property name="accessTokenUri" value="${ZAC_UAA_TOKEN_URL:${ZAC_UAA_URL}/oauth/token}" />
-                <property name="clientId" value="${zacClientId}" />
-                <property name="clientSecret" value="${zacClientSecret}" />
-            </bean>
-        </constructor-arg>
-        <property name="requestFactory" ref="httpRequestFactory"/>
-    </bean>
-
     <!-- Begin constructing Apache HttpClient request factory for REST template. -->
 
     <bean id="httpRequestFactory"
-        class="org.springframework.http.client.HttpComponentsClientHttpRequestFactory">
-        <constructor-arg ref="httpClient" />
+          class="org.springframework.http.client.HttpComponentsClientHttpRequestFactory">
```

Comment:
These cannot be moved under predix profile because we utilize them for the acsTemplate in the acceptance-test-spring-context.xml